### ISSUE TITLE
Unable to use field name starting with "set"

### ISSUE THREAD

**Issue by ubaierbhat** [agrosner/DBFlow#1380]
DBFlow Version: 4+
Issue Kind : Bug;
Description: 

Weird issue with field names starting with "set" … eg. ”settlement”, "setting". This causes compilation issues: 

`Error:(118, 10) error: cannot find symbol method setting(String)`

```
@Table(database = AppDatabase.class)
public class Address  {

    @PrimaryKey
    private Long id;

    @Column
    private String setting;

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getSetting() {
        return setting;
    }

    public void setSetting(String setting) {
        this.setting = setting;
    }
}
```

Generated class
```
public final class Address_Table extends ModelAdapter<Address> {
  …

  @Override
  public final void loadFromCursor(FlowCursor cursor, Address model) {
    model.setId(cursor.getLongOrDefault("id", null));
    model.setting(cursor.getStringOrDefault("setting")); \\\ oops .. setter name 
  }

  …
}
```

Quick fix is to keep the field package private .  I guess something to do with kotlin?

**Comment by ubaierbhat** [comment]
Looks like we have same issue with a field name starting with "get". `Error:(179, 35) error: cannot find symbol method getaway()`
 
another option other than making the field package private is to specify the getter or setter name;

```
    @Column(setterName = "setSetting")
    private String setting;
```

If the name starts with get (less likely but still) 

```
    @Column(getterName = "getGetaway")
    private String getaway;
```

**Comment by agrosner** [comment]
I will change up logic for 4.1.0. I realize error of the check. It should do a "search" to find appropriate getter and setter on class instead of assuming its one way or other. 
```
 if (useIsForPrivateBooleans && !propertyName.startsWith("is", ignoreCase = true)) {
                    "is" + propertyName.capitalize()
                } else if (!useIsForPrivateBooleans && !propertyName.startsWith("get", ignoreCase = true)) {
                    "get" + propertyName.capitalize()
                } else propertyName.lower()
```

**Comment by agrosner** [comment]
fixed in `5.0.0-alpha2`
